astropy/wcs/utils.py

```diff
@@ -163,6 +216,28 @@ def _celestial_frame_to_wcs_builtin(frame, projection="TAN"):
         ycoord = "TLAT"
         wcs.wcs.radesys = "ITRS"
         wcs.wcs.dateobs = frame.obstime.utc.isot
+    elif hasattr(frame, "name") and frame.name in solar_system_obj_dict.values():
+        xcoord = frame.name[:2].upper().replace("MO", "SE") + "LN"
+        ycoord = frame.name[:2].upper().replace("MO", "SE") + "LT"
+        if issubclass(frame.representation_type, BaseGeodeticRepresentation):
+            wcs.wcs.name = "Planetographic Body-Fixed"
+        elif issubclass(frame.representation_type, BaseBodycentricRepresentation):
+            wcs.wcs.name = "Bodycentric Body-Fixed"
+        if Version(_wcs.__version__) >= Version("8.0"):
```

Comment:
If we do need a version check, then here it would be fine to do `if WCSLIB_LT_8: raise ...`